src/test/java/com/coherentsolutions/training/aqa/java/web/nikonova/objects/BaseTest.java

```diff
@@ -0,0 +1,36 @@
+package com.coherentsolutions.training.aqa.java.web.nikonova.objects;
+
+import com.coherentsolutions.training.aqa.java.web.nikonova.browsers.BrowsersProperties;
+import com.coherentsolutions.training.aqa.java.web.nikonova.browsers.BrowsersSettings;
+import org.openqa.selenium.WebDriver;
+
+import org.testng.annotations.*;
+
+import java.time.Duration;
+
+
+public class BaseTest {
+
+    protected static final String URL = "http://automationpractice.com/index.php?controller=authentication&back=my-account";
+    protected WebDriver driver = BrowsersSettings.getDriver();
+
+    @BeforeClass
+    public void setupWebDriver() throws Exception {
+        System.setProperty("webdriver.chrome.driver",
+                "C:\\Users\\SvetlanaNikonova\\chromedriver.exe");
```

Comment:
Please do not use absolute paths, only related. Just imagine situation, someone from your teammates pulling your code from repo to local machine and he will not be able to start these tests because he doesn't have such package: C:\\Users\\SvetlanaNikonova\\chromedriver.ex